src/autopas/containers/verletClusterLists/VerletClusterListsRebuilder.h

```diff
@@ -85,7 +84,7 @@ class VerletClusterListsRebuilder {
       numParticles += vector.size();
     }
 
-    auto boxSize = utils::ArrayMath::sub(_boxMax, _boxMin);
+    auto boxSize = utils::ArrayMath::sub(_haloBoxMax, _haloBoxMin);
```

Comment:
```suggestion
    auto boxSizeWithHalo = utils::ArrayMath::sub(_haloBoxMax, _haloBoxMin);
```
or ``haloBoxSize` (<-however, I'm not too convinced about this one)